share/man/nitc.md

```diff
@@ -223,6 +223,11 @@ See the documentation of these specific modules for details.
 
     Currently, this only affect the android platform.
 
+`-g`, `--debug`
+:   Compile in debug mode.
+
+    Currently removes gcc optimizations.
```

Comment:
Maybe also suggest to use `--hardening` and/or `NIT_GC_OPTION` as aditional tools for debuging